raiden-dapp/tests/unit/components/account/account-content.spec.ts

```diff
@@ -200,4 +200,36 @@ describe('AccountContent.vue', () => {
       }),
     );
   });
+
+  test('displays "Disconnect" button when connected', async () => {
+    const wrapper = await createWrapper(
+      undefined,
+      undefined,
+      undefined,
+      undefined,
+      false,
+      undefined,
+    );
+    const disconnectButton = wrapper.find('.account-content__disconnect__button');
+
+    expect(disconnectButton.exists()).toBe(false);
```

Comment:
Is this a mistake? I'm not sure if I need another coffee or if I'm just stupid. :laughing: 
I mean this test works because you explicitly create the wrapper with connection set to `false` but the test case title states something different.

Reply:
Yeah, I mixed up the test names, will correct it. :)